bin/spf.ml

```diff
@@ -18,11 +18,12 @@ let ns_check ~domain spf =
       Dns_client_lwt.get_resource_record dns key domain_name
   end in
   let he = Happy_eyeballs_lwt.create () in
-  let dns = Dns_client_lwt.create he in
-  Uspf_lwt.get ~domain dns (module DNS) >>= function
+  let _dns = Dns_client_lwt.create he in
+  assert false
+(* Uspf_lwt.get ~domain dns (module DNS) >>= function
```

Comment:
the assert false looks a bit fishy, is this intentional?

Reply:
Yes, I need to implement a thing on `uspf` but it not really important for server implementations.